websockets/py36/protocol.py

```diff
@@ -0,0 +1,3 @@
+async def __aiter__(self):
+    while True:
+        yield await self.recv()
```

Comment:
Good points, agreed. This was just a POC to get the discussion going, so I hadn't yet given thought myself to proper error handling / cleanup, etc.